app/controllers/time_controller.rb

```diff
@@ -0,0 +1,6 @@
+class TimeController < ApplicationController
+  def now
+    @time = Time.now
```

Comment:
You don't use `@time` in the view, so these two lines could be `@time_formatted = Time.zone.now.strftime(...)`